src/preset.ts

```diff
@@ -0,0 +1,86 @@
+const commonOptions: Fig.Option[] = [
+  {
+    name: "--no-interaction",
+    description: "Disable interactions",
+  },
+  {
+    name: "--debug",
+    description: "Display debug information instead of standard output",
+  },
+  {
+    name: "--silent",
+    description: "Do not print anything",
+  },
+];
+
+const applyOptions: Fig.Option[] = [
+  {
+    name: ["--path", "-p"],
+    description: "Path to a sub-directory in which to look for a preset",
+  },
```

Comment:
Should this take a path argument?
```suggestion
  {
    name: ["--path", "-p"],
    description: "Path to a sub-directory in which to look for a preset",
    args: {
      name: "path",
      template: "folders",
    },
  },
```

Reply:
No, because that would be a subdirectory of something we don't have access to (eg. the directory of the cloned repository)